src/v0/destinations/klaviyo/util.js

```diff
@@ -305,6 +309,28 @@ const batchSubscribeEvents = (subscribeRespList) => {
   return batchedResponseList;
 };
 
+const addSubscribeFlagToTraits = (traitsInfo) => {
+  let traits = traitsInfo;
+  if (!isDefinedAndNotNull(traits)) {
+    return traits;
+  }
+  // check if properties already contains subscribe flag
+
+  if (traits.properties) {
+    if (traits.properties.subscribe === undefined) {
+      traits.properties.subscribe = true;
+    } else {
+      // return if subscribe flag is already present
+      return traits;
+    }
+  } else {
+    traits = {
+      properties: { subscribe: true },
```

Comment:
will this be a breaking change for existing connections?

Reply:
if they were not sending properties.subscribe flag but they configured listId in UI for VDM this can become a breaking change we need to verify such customers once